index.js

```diff
@@ -13,10 +13,38 @@ const storagePath = path.join('elm-stuff', 'gitdeps');
 
 const args = process.argv.slice(2);
 
+const helpMsg =
+`use 'elm-git-install' with no arguments to install the dependencies in you 'elm-git.json' file
```

Comment:
typo: s/you/your/

(I'm not a maintainer, but happened to look at this.)

Reply:
Fixed 👍